lrt-pegs/balancer/main.py

```diff
@@ -0,0 +1,68 @@
+from web3 import Web3
+from dotenv import load_dotenv
+import os, json, requests
+
+load_dotenv()
+
+peg_threshold = 0.05 # 5%
```

Comment:
Is this used for something?

Reply:
yep it will be somethignlike 70% in prod